tests/scripts/github-action-helper.sh

```diff
@@ -424,6 +424,32 @@ EOF
   diff 1M.dat 1M-get.dat
 }
 
+function write_object_to_cluster2_read_from_cluster1() {
```

Comment:
Instead of duplicating the function, can we call the common function and pass different parameters? Less code to maintain in the long term.

Reply:
@jhoblitt Is the only difference between these methods which is the cluster 1 or 2 ip address? If so, would be great to refactor this to avoid duplication.

Reply:
@travisn Yes, I'm actively working on DRYing it up right now.

Reply:
![image](https://user-images.githubusercontent.com/948901/213514757-d9a7dad8-6098-4a4b-b7ec-6a2449409d80.png)